apps/explorer/src/ui/RingChart/index.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+export { RingChart as default, type RingChartProps } from './RingChart';
```

Comment:
Can you just move these to index?

Reply:
Also prefer named export.

Reply:
Updated. If we move this into `index`, then we will get 2 different lines for importing `RingChart` and `RingChartLegend` which are from the same folder. Though we could move them into 1 big file but that would be a 200 lines file, unless you think it is better to have everything in 1 file instead of index exporting.

Reply:
200 lines doesn't scare me